microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -452,3 +452,12 @@ Issued: 19 September 2024
 The {product-title} release 4.15.33 is now available. The list of bug fixes that are included in the update is attached to the link:https://access.redhat.com/errata/RHBA-2024:6686[RHBA-2024:6686] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:6685[RHSA-2024:6685] advisory.
 
 For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-15-34-dp_{context}"]
+=== RHBA-2024:6820 - {microshift-short} 4.15.34 bug fix and enhancement advisory
+
+Issued: 24 September 2024
```

Comment:
I think this should be 25 September 2024?

Reply:
You're right, I must have been looking at the other one.